traja/tests/test_dataset.py

```diff
@@ -3,7 +3,8 @@
 import pytest
 
 from traja.dataset import dataset
-from traja.dataset.pituitary_gland import create_latin_hypercube_sampled_pituitary_df
+from traja.dataset.pituitary_gland import create_latin_hypercube_sampled_pituitary_df, \
+    pituitary_ori_ode_parameters_Isk_Ibk_Ikir_Icat_Ia_Inav, generate_pituitary_dataset
```

Comment:
Hi @WolfByttner Please rename these methods `pituitary_ori_ode_parameters_Isk_Ibk_Ikir_Icat_Ia_Inav`. For example something like `get_params(*list(pnames),**kwargs)` would be ideal. More than that travis CI stuck at those methods for some weird reasons.